### DISCUSSION TITLE
5.1如何设置namesrv的日志路径

### DISCUSSION THREAD

**Discussion by Hoxxx** [apache/rocketmq#7207] [Q&A] [answered]
版本5.1.3，想知道有没有什么参数可以指定namesrv的日志路径？
现在只启动namesrv，会在/root/logs/rocketmqlogs下生成大量日志文件，日志级别 INFO
启动方式：
nohup sh /data/rocketmq/bin/mqnamesrv -c /data/rocketmq/conf/namesrv.conf > /data/rocketmq/logs/mqnamesrv.log 2>&1

**Comment by Hoxxx** [comment, marked as answer]
没事了，改conf下的xml文件
P.S：吐槽下，能否将日志路径合并到一个文件。。

**Reply by francisoliverlee** [reply]
you can update namesv log config by edit conf/logback_namesrv.xml
